### PR TITLE
Migrate to Python 3

### PR DESCRIPTION
**Pull request by kosmos342**

### CODE REVIEW

File: yolapi/pypi/models.py [hunk 1]
```diff
@@ -127,8 +127,8 @@ def gc(self):
         if not self.distributions.exists():
             self.delete()
 
-    def __unicode__(self):
-        return u'%s %s' % (self.package.name, self.version)
+    def __str__(self):
+        return f'{self.package.name} {self.version}'
```
Comment (snitch) [inline yolapi/pypi/models.py:131]: E999 SyntaxError: invalid syntax

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Review by RayeN — APPROVED** [review]

**Pull request merged**